src/game/client/draw/w3dmodeldraw.cpp

```diff
@@ -1879,7 +1879,7 @@ bool W3DModelDraw::Update_Bones_For_Client_Particle_Systems()
     return true;
 }
 
-char *s_terrainDecalTextureName[] = { "TERRAIN_DECAL_DEMORALIZED_OBSOLETE",
+const char *const s_terrainDecalTextureName[] = { "TERRAIN_DECAL_DEMORALIZED_OBSOLETE",

```

Comment:
Could make this constexpr to further cement it's fixed value ness

Reply:
As far as I am aware `constexpr` or not makes no practical difference here.